pkgs/by-name/wh/whoami/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "whoami";
+  version = "1.10.3";
+
+  src = fetchFromGitHub {
+    owner = "traefik";
+    repo = "whoami";
+    rev = "v${version}";
+    hash = "sha256-wzxgmysqn4aWAZEaMjMwHdHLe4UZ4nwdNFJw5X7fuKQ=";
+  };
+
+  vendorHash = "sha256-qDfkYIAymkFUtbKka9OLoYjT+S9KhOra2UtOvhoz5Mw=";
+
+  ldflags = [ "-s" ];
+
+  CGO_ENABLED = 0;
+
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
Remove `with lib;`.

We try to avoid as much as possible the use of `with` in front of an attribute set.